spi-impl/series/series-dao-webapp/src/main/webapp/WEB-INF/spring/dispatcher-servlet.xml

```diff
@@ -48,11 +48,11 @@
     <bean class="org.springframework.web.servlet.view.ContentNegotiatingViewResolver">
         <property name="viewResolvers">
             <list>
-<!--                 <bean class="org.springframework.web.servlet.view.BeanNameViewResolver" /> -->
-<!--                 <bean class="org.springframework.web.servlet.view.InternalResourceViewResolver"> -->
-<!--                     <property name="prefix" value="/WEB-INF/jsp/" /> -->
-<!--                     <property name="suffix" value=".jsp" /> -->
-<!--                 </bean> -->
+                 <bean class="org.springframework.web.servlet.view.BeanNameViewResolver" />
+                 <bean class="org.springframework.web.servlet.view.InternalResourceViewResolver">
+                     <property name="prefix" value="/" />
```

Comment:
use a prefix different from `/` just to separate API-related content from stuff the actual webapp ships.